modules/rtticketedit.php

```diff
@@ -150,13 +150,18 @@
                 break;
             case 'resolve':
                 $ticket = $LMS->GetTicketContents($id);
-
-                if ($ticket['state'] == RT_RESOLVED) {
-                    $SESSION->redirect('?m=rtticketview&id=' . $id);
+                if (ConfigHelper::checkConfig('phpui.helpdesk_block_ticket_close_with_open_events') && !empty($ticket['openeventcount'])) {
+                    die(trans("Ticket have open assigned events!"));
```

Comment:
To jest komunikat błędu, gdy ktoś spróbuje oszukać?
Co w sytuacji, gdy formularz wyświetlający listę zgłoszeń wyświetli się zanim znikną warunki pozwalające na zamknięcie zgłoszenia, np. pojawi się nowe zdarzenie powiązane w trybie otwartym.
Wówczas biały ekran i komunikat?

Reply:
> To jest komunikat błędu, gdy ktoś spróbuje oszukać?

Dokładnie tak.

> Co w sytuacji, gdy formularz wyświetlający listę zgłoszeń wyświetli się zanim znikną warunki pozwalające na zamknięcie zgłoszenia, np. pojawi się nowe zdarzenie powiązane w trybie otwartym.
Wówczas biały ekran i komunikat?

Nie umiem jeszcze dobrze w xajax. Faktycznie to dobre miejsce na to.

Reply:
Może zaakceptujesz "Aa is" (bo to rozwiązuje problem) a xajax dorobię jak się nauczę?